benchmarks/populate_into_db.py

```diff
@@ -0,0 +1,160 @@
+import argparse
+import os
+import sys
+
+import pandas as pd
+import psycopg2
+import psycopg2.extras
+
+
+FINAL_CSV_FILENAME = "collated_results.csv"
+# https://github.com/huggingface/transformers/blob/593e29c5e2a9b17baec010e8dc7c1431fed6e841/benchmark/init_db.sql#L27
+BENCHMARKS_TABLE_NAME = "benchmarks"
+MEASUREMENTS_TABLE_NAME = "model_measurements"
+
+
+def _init_benchmark(conn, branch, commit_id, commit_msg):
+    metadata = {"metadata": f"diffusers-{commit_id}"}
```

Comment:
What is the purpose of the field? Doesn't bring much value imo, you can remove it. This field is more to store extra info, in transformers we store gpu name and model id for instance.

Reply:
Indeed. Can we keep it null?

Reply:
The field isn't nullable 😅 
Guess you can put an empty dict though.